.env.example

```diff
@@ -1,3 +0,0 @@
-MAPS_API_KEY=<your_google_maps_api_key>
-HTTP_ENDPOINT=<server's_HTTP_endpoint>(for example: https://beacon.aadibajpai.com/graphql)
-WEBSOCKET_ENDPOINT=<server's_websocket_endpoint>(for example: wss://beacon.aadibajpai.com/subscriptions)
```

Comment:
Revert this change. This must not be removed

Reply:
I have deleted this file.

Reply:
This file must not be deleted, restore it. How would beginners setup the project without .env keys

Reply:
I am really sorry.I have added the file again.